pkgs/applications/misc/uni/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "uni";
+  version = "2.5.1";
+
+  src = fetchFromGitHub {
+    owner = "arp242";
+    repo = "uni";
+    rev = "v${version}";
+    sha256 = "kWiglMuJdcD7z2MDfz1MbItB8r9BJ7LUqfPfJa8QkLA=";
+  };
+
+  vendorSha256 = "6HNFCUSJA6oduCx/SCUQQeCHGS7ohaWRunixdwMurBw=";
+
+  ldflags = [ "-X main.version=${version}" ];
```

Comment:
Missing -s -w

Reply:
Is this documented somewhere that this should be added? It's in the [ldflags example](https://nixos.org/manual/nixpkgs/stable/#sec-language-go) in the nixpkgs manual, but it doesn't mention any requirement of these flags. I would also expect this to be stripped anyway in the `fixupPhase`?

Reply:
> Is this documented somewhere that this should be added?

maybe IDK



> It's in the [ldflags example](https://nixos.org/manual/nixpkgs/stable/#sec-language-go) in the nixpkgs manual

You always want to look at the unstable doc or the one in nixpkgs. https://nixos.org/manual/nixpkgs/unstable/#sec-language-go

> it doesn't mention any requirement of these flags.

If you are building go binaries for production you always want to set those flags.



> I would also expect this to be stripped anyway in the `fixupPhase`?

It is not the exact same stripping. AFAIK also removes some extra symbols strip can't remove.